modules/accessing-inference-endpoint-for-model-deployed-on-single-model-serving-platform.adoc

```diff
@@ -21,174 +21,36 @@ endif::[]
 . From the {productname-short} dashboard, click *Model Serving*.
 +
 The inference endpoint for the model is shown in the *Inference endpoint* field.
-. Depending on what action you want to perform with the model (and if the model supports that action), copy the inference endpoint shown and then add one of the following paths to the end of the URL:
-+
---
-*Caikit TGIS ServingRuntime for KServe*
-
-* `:443/api/v1/task/text-generation`
-* `:443/api/v1/task/server-streaming-text-generation`
-// * `:443/api/v1/task/text-classification`
-// * `:443/api/v1/task/token-classification`
-
-*Caikit Standalone ServingRuntime for KServe*
+. Depending on what action you want to perform with the model (and if the model supports that action), copy the inference endpoint shown and then add a path to the end of the URL.
 
-.REST endpoints
-* `/api/v1/task/embedding`
-* `/api/v1/task/embedding-tasks`
-* `/api/v1/task/sentence-similarity`
-* `/api/v1/task/sentence-similarity-tasks`
-* `/api/v1/task/rerank`
-* `/api/v1/task/rerank-tasks`
-
-.gRPC endpoints
-* `:443 caikit.runtime.Nlp.NlpService/EmbeddingTaskPredict`
-* `:443 caikit.runtime.Nlp.NlpService/EmbeddingTasksPredict`
-* `:443 caikit.runtime.Nlp.NlpService/SentenceSimilarityTaskPredict`
-* `:443 caikit.runtime.Nlp.NlpService/SentenceSimilarityTasksPredict`
-* `:443 caikit.runtime.Nlp.NlpService/RerankTaskPredict`
-* `:443 caikit.runtime.Nlp.NlpService/RerankTasksPredict`
-+
 ifdef::upstream[]
-NOTE: By default, the Caikit Standalone Runtime exposes REST endpoints for use. To use gRPC protocol, manually deploy a custom Caikit Standalone ServingRuntime. For more information, see link:{odhdocshome}/serving-models/#adding-a-custom-model-serving-runtime-for-the-single-model-serving-platform_serving-large-models[Adding a custom model-serving runtime for the single-model serving platform]. 
-
-An example manifest is available in the link:https://github.com/opendatahub-io/caikit-tgis-serving/blob/main/demo/kserve/custom-manifests/caikit/caikit-standalone/caikit-standalone-servingruntime-grpc.yaml[caikit-tgis-serving GitHub repository^].
+[NOTE]
+====
+For a list of paths to use with the supported runtimes, see link:{odhdocshome}/serving-models/#inference-endpoints_serving-large-models[Inference endpoints]
```

Comment:
Missing period (.) at the end of the sentence, after the link.

Reply:
```suggestion
For a list of paths to use with the supported runtimes, see link:{odhdocshome}/serving-models/#inference-endpoints_serving-large-models[Inference endpoints].
```